### PR TITLE
Bugfix/Readme Update

### PR DESCRIPTION
**Pull request by sampoearas**
- Bugfix for missing close tag
- Readme History Updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ This project and the following release notes have been migrated from the old Ara
 
 Release | Notes
 --------|--------
+[v5.0.0](https://github.com/ArasLabs/my-recent-items/releases/tag/v5.0.0) | Bugfix and method optomization. Readme History Updated.
```
Comment (EliJDonahue) [inline README.md]: What's the method optimization? Is this PR missing a commit(s)?
```suggestion
[v5.0.0](https://github.com/ArasLabs/my-recent-items/releases/tag/v5.0.0) | Bugfix and method optimization. Readme History Updated.
```

**Review by EliJDonahue — CHANGES_REQUESTED** [review]
I'd probably just make this release v4.1.1 since it only includes a bug fix/doc update and it's still compatible with previous releases of Aras.

File: README.md (hunk 1 above)
Comment (sampoearas) [reply README.md]: Method optimization was Angela's PR. I'm combining my bugfix with her PR in a single release, thus 5.0 since there was a code change

Comment (EliJDonahue) [reply README.md]: Ah! Makes sense. Just fix the spelling typo then and this is good to go.

**Comment by sampoearas** [comment]
changes commited

**Pull request merged**